supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendSingletonActor.scala

```diff
@@ -3,19 +3,21 @@ package cromwell.backend.impl.jes
 import akka.actor.{Actor, ActorLogging, Props}
 import cromwell.core.Dispatcher.BackendDispatcher
 import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager
-import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.DoPoll
+import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.JesApiQueryManagerRequest
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Positive
 
-final case class JesBackendSingletonActor(qps: Int) extends Actor with ActorLogging {
+final case class JesBackendSingletonActor(qps: Int Refined Positive) extends Actor with ActorLogging {
```

Comment:
Oh I was just commenting that my normal objections don't apply here so if by any chance you had originally said "I want to make a `type` but mean @geoffjentry would hate it!" to feel free. Although who am I kidding, that never stops anyone ;)